packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/Buy/template.success.js

```diff
@@ -70,78 +75,82 @@ const CoinifyBuy = props => {
 
   const limits = service.getLimits(profile._limits, defaultCurrency)
 
-  if (step === 'checkout') {
+  if (equals(step, 'checkout')) {
     return (
-      <Stepper initialStep={0}>
-        <StepView step={0}>
-          <CheckoutWrapper>
-            <LeftContainer>
-              <OrderCheckout
-                quoteR={buyQuoteR}
-                onFetchQuote={fetchBuyQuote}
-                limits={limits.buy}
-                type={'buy'}
-                reason={'has_remaining'} // placeholder for now - coinify does not require a reason
-                defaultCurrency={defaultCurrency}
-                symbol={symbol}
-                checkoutBusy={checkoutBusy}
-                setMax={setMax}
-                setMin={setMin}
-                increaseLimit={handleKycAction}
-              />
-            </LeftContainer>
-            <RightContainer>
-              {activeSubscriptions.length > 0 ? (
-                <NextSubscription
-                  subscriptions={subscriptions}
-                  trades={trades.filter(t => t.tradeSubscriptionId)}
-                  manageOrder={() => changeTab('order_history')}
-                />
-              ) : null}
-              {path(['state'], kyc) ? (
-                <KYCNotification
-                  kyc={kyc}
-                  limits={limits.buy}
-                  symbol={symbol}
-                  onTrigger={kyc => handleKycAction(kyc)}
-                  canTrade={canTrade}
-                />
-              ) : null}
-            </RightContainer>
-          </CheckoutWrapper>
-        </StepView>
-        <StepView step={1}>
-          <Payment />
-        </StepView>
-        <StepView step={2}>
-          <CheckoutWrapper>
-            <OrderDetails
-              quoteR={buyQuoteR}
-              onRefreshQuote={refreshQuote}
-              type={'buy'}
-              medium={paymentMedium}
+      <CheckoutWrapper>
+        <LeftContainer>
+          <OrderCheckout
+            checkoutBusy={checkoutBusy}
+            coinifyNextCheckoutStep={coinifyNextCheckoutStep}
+            countryCode={countryCode}
+            defaultCurrency={defaultCurrency}
+            increaseLimit={handleKycAction}
+            limits={limits.buy}
+            onFetchQuote={fetchBuyQuote}
+            openRecurringConfirmModal={openRecurringConfirmModal}
+            quoteR={buyQuoteR}
+            reason={'has_remaining'}
+            setMax={setMax}
+            setMin={setMin}
+            showRecurringBuy={showRecurringBuy}
+            showRecurringModal={showRecurringModal}
+            symbol={symbol}
+            type={'buy'}
+          />
+        </LeftContainer>
+        <RightContainer>
+          {activeSubscriptions.length > 0 ? (
+            <NextSubscription
+              subscriptions={subscriptions}
+              trades={trades.filter(t => t.tradeSubscriptionId)}
```

Comment:
`filter(prop('tradeSubscriptionId'))` could help avoid null pointer and remove 1-digit variable